app/code/community/Payone/Core/Model/Service/InitializeConfig.php

```diff
@@ -55,21 +55,24 @@ class Payone_Core_Model_Service_InitializeConfig
      * @param int $storeId
      * @return Payone_Core_Model_Config_Interface
      */
-    public function execute($storeId = null)
+    public function execute($storeId = null, $useCache = true)
     {
         $this->setStoreId($storeId);
 
         $helperRegistry = $this->helperRegistry();
```

Comment:
As this line and the following one are not used anymore out of the if-statement, maybe we could move them inside the statement as well ?

Reply:
Good point. I moved helperRegistry and registryKey inside the if statement